Integrations/TinkoffIntegrationLib/TinkoffInstrumentAdapter.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Tinkoff.Trading.OpenApi.Models;
+
+namespace TinkoffIntegrationLib
+{
+    class TinkoffInstrumentAdapter : IMarketInstrument
+    {
+        MarketInstrument tinkoffInstrument;
+        Orderbook tinkoffOrderbook;
+
+        /*
+         * If I will pass to ctor tinkoff context,
```

Comment:
What does it mean ctor?

Reply:
Constructor i guess

Reply:
Try to write ctor in Visual Studio and press TAB twice